rmgpy/cantherm/input.py

```diff
@@ -388,27 +411,111 @@ def loadInputFile(path):
+    level_of_theory = local_context.get('levelOfTheory', '')
+    author = local_context.get('author', '')
     if 'frequencyScaleFactor' not in local_context:
         logging.debug('Assigning a frequencyScaleFactor according to the modelChemistry...')
-        frequencyScaleFactor = assign_frequency_scale_factor(modelChemistry)
+        frequency_scale_factor = assign_frequency_scale_factor(model_chemistry)
     else:
-        frequencyScaleFactor = local_context.get('frequencyScaleFactor')
-    useHinderedRotors = local_context.get('useHinderedRotors', True)
-    useAtomCorrections = local_context.get('useAtomCorrections', True)
-    useBondCorrections = local_context.get('useBondCorrections', False)
-    atomEnergies = local_context.get('atomEnergies', None)
+        frequency_scale_factor = local_context.get('frequencyScaleFactor')
+    use_hindered_rotors = local_context.get('useHinderedRotors', True)
+    use_atom_corrections = local_context.get('useAtomCorrections', True)
+    use_bond_corrections = local_context.get('useBondCorrections', False)
+    atom_energies = local_context.get('atomEnergies', None)
     
     directory = os.path.dirname(path)
     
     for job in jobList:
         if isinstance(job, StatMechJob):
             job.path = os.path.join(directory, job.path)
-            job.modelChemistry = modelChemistry.lower()
-            job.frequencyScaleFactor = frequencyScaleFactor
-            job.includeHinderedRotors = useHinderedRotors
-            job.applyAtomEnergyCorrections = useAtomCorrections
-            job.applyBondEnergyCorrections = useBondCorrections
-            job.atomEnergies = atomEnergies
+            job.modelChemistry = model_chemistry.lower()
+            job.frequencyScaleFactor = frequency_scale_factor
+            job.includeHinderedRotors = use_hindered_rotors
+            job.applyAtomEnergyCorrections = use_atom_corrections
+            job.applyBondEnergyCorrections = use_bond_corrections
+            job.atomEnergies = atom_energies
+        if isinstance(job, ThermoJob):
+            job.cantherm_species.species_dictionary['author'] = author
+            job.cantherm_species.species_dictionary['level_of_theory'] = level_of_theory
+            job.cantherm_species.species_dictionary['model_chemistry'] = model_chemistry
+            job.cantherm_species.species_dictionary['frequency_scale_factor'] = frequency_scale_factor
+            job.cantherm_species.species_dictionary['use_hindered_rotors'] = use_hindered_rotors
+            job.cantherm_species.species_dictionary['use_atom_corrections'] = use_atom_corrections
+            job.cantherm_species.species_dictionary['use_bond_corrections'] = use_bond_corrections
+            job.cantherm_species.species_dictionary['atom_energies'] = atom_energies
     
     return jobList
+
+
+def load_species_from_database_file(job):
+    """
+    Load the species with all statMech data from the .yml file
+    """
+    global jobList
```

Comment:
It doesn't seem like you use jobList in this function.

Reply:
removed